openwisp_ipam/models.py

```diff
@@ -17,11 +17,11 @@ def request_ip(self, options=None):
         ip = self.get_first_available_ip()
         if not ip:
             return None
-        ip_address = load_model('openwisp_ipam', 'IpAddress')(
-                                ip_address=ip,
-                                subnet=self,
-                                organization=self.organization,
-                                **options)
+        ip_address = load_model('openwisp_ipam',
+                                'IpAddress')(ip_address=ip,
+                                             subnet=self,
+                                             organization=self.organization,
+                                             **options)
```

Comment:
revert this change as it's not related to this patch and introducing unrelated changes is a bad practice

Reply:
i always get errors with flake8 and i modified this files. I'll rollback this file.